erts/emulator/beam/erl_db_tree.c

```diff
@@ -3533,11 +3533,11 @@ void* db_eterm_to_dbterm_tree_common(int compress, int keypos, Eterm obj)
     return term;
 }
 
-void* db_dbterm_list_prepend_tree_common(void *list, void *db_term)
+void* db_dbterm_list_append_tree_common(void *last_term, void *db_term)
```

Comment:
Not sure if/where this code is even used, but have modified it to be consistent with `erl_db_hash` for consistency.

Reply:
It's for `ordered_set` tables.